learning/features/assetlib/what_is_assetlib.rst

```diff
@@ -0,0 +1,20 @@
+.. _doc_what_is_assetlib:
+
+About the Asset Library
+=======================
+
+The Godot Asset Library, otherwise known as the AssetLib, is a repository of
+user-submitted Godot addons, scripts and other resources, collectively referred
+to as assets, available for all Godot users for download and use in their own
+Godot projects. It is available on the `Godot website <http://godotengine.org/asset-library>`_,
+and is also accessible directly from within the engine's editor interface.
+
+While on the surface the Asset Library might look and function similar to asset
+stores available for other engines (such as Unity's Asset Store, or Unreal
+Engine's Marketplace), in practice it is a lot more similar to GNU Emacs' ELPA,
+or Mozilla Firefox's Add-ons repository - the AssetLib does not support selling
+paid assets (though in the future, a third-party asset store that allows this
+might surface), and while not technically enforced, users are encouraged to 
+distribute their assets under a free and open-source license, such as the MIT 
+license (the same kind of license that the Godot Engine itself is licensed as),
```

Comment:
4 parens in 1 sentence. NB: I count dashes as parens. Use 4-5 shorter sentences instead